core/src/main/scala/cromwell/util/JsonEditor.scala

```diff
@@ -83,7 +83,17 @@ object JsonEditor {
 
   def outputs(json: Json): Json = includeJson(json, NonEmptyList.of("outputs")) |> (excludeJson(_, NonEmptyList.one("calls")))
 
-  def logs(json: Json): Json = includeJson(json, NonEmptyList.of("stdout", "stderr", "backendLogs"))
+  def logs(json: Json): ErrorOr[Json] = {
+    // Deletes all subworkflows found in the input by always returning `None`.
+    def deleteSubworkflows(o: JsonObject, j: Json): Option[ErrorOr[Json]] = None
+    for {
+      wf <- updateWorkflowCallsJson(json, deleteSubworkflows)
+      // exclude outputs and inputs since variables can be named anything including internally reserved words like
+      // `stdout` and `stderr` which would be erroneously included among the logs.
+      inc = excludeJson(wf, NonEmptyList.of("outputs", "inputs")) |>
+        (includeJson(_, NonEmptyList.of("shardIndex", "attempt", "stdout", "stderr", "backendLogs")))
```

Comment:
Could make sense to extract the list of `"shardIndex", "attempt", "stdout", "stderr", "backendLogs"` into a variable.